obspy/taup/velocity_model.py

```diff
@@ -484,6 +485,125 @@ def readTVelFile(cls, filename):
                              cls.default_cmb, cls.default_iocb, 0,
                              maxRadius, True, layers)
 
+    @classmethod
+    def readNDFile(cls, filename):
```

Comment:
Can you rename this to `read_nd_file()` to suite the rest of ObsPy? I know `obspy.taup` does not adhere to it very strictly but new stuff should probably try to do it.